plugins/by-name/conform-nvim/formatter-packages.nix

```
@@ -0,0 +1,146 @@
+    trim_whitespace = null; # Conform native formatter
+
+    auto_optional = sType.unpackaged;
+    bake = sType.unpackaged;
+    blue = sType.unpackaged;
+    bpfmt = sType.unpackaged;
+    bsfmt = sType.unpackaged;
+    caramel_fmt = sType.unpackaged;
+    crlfmt = sType.unpackaged;
+    darker = sType.unpackaged;
+    dcm_fix = sType.unpackaged;
+    dcm_format = sType.unpackaged;
+    easy-coding-standard = sType.unpackaged;
+    findent = sType.unpackaged;
+    ghokin = sType.unpackaged;
+    gluon_fmt = sType.unpackaged;
+    grain_format = sType.unpackaged;
+    hledger-fmt = sType.unpackaged;
+    imba_fmt = sType.unpackaged;
+    janet-format = sType.unpackaged;
+    liquidsoap-prettier = sType.unpackaged;
+    llf = sType.unpackaged;
+    markdown-toc = sType.unpackaged;
+    markdownfmt = sType.unpackaged;
+    mdslw = sType.unpackaged;
+    mojo_format = sType.unpackaged;
+    nomad_fmt = sType.unpackaged;
+    npm-groovy-lint = sType.unpackaged;
+    packer_fmt = sType.unpackaged;
+    pangu = sType.unpackaged;
+    perlimports = sType.unpackaged;
+    pint = sType.unpackaged;
+    purs-tidy = sType.unpackaged;
+    pycln = sType.unpackaged;
+    pyink = sType.unpackaged;
+    pymarkdownlnt = sType.unpackaged;
+    reformat-gherkin = sType.unpackaged;
+    rescript-format = sType.unpackaged;
+    runic = sType.unpackaged;
+    spotless_gradle = sType.unpackaged;
+    spotless_maven = sType.unpackaged;
+    standard-clj = sType.unpackaged;
+    standardjs = sType.unpackaged;
+    tlint = sType.unpackaged;
+    twig-cs-fixer = sType.unpackaged;
+    vsg = sType.unpackaged;
+    ziggy = sType.unpackaged;
+    ziggy_schema = sType.unpackaged;
+
+    inherit (python313Packages) autopep8;
+    awk = gawk;
+    bean-format = beancount;
+    biome-check = biome;
+    biome-organize-imports = biome;
+    cabal_fmt = haskellPackages.cabal-fmt;
+    clang_format = clang-tools;
+    clang-format = clang-tools;
+    cmake_format = cmake-format;
+    css_beautify = nodePackages.js-beautify;
+    cue_fmt = cue;
+    dart_format = dart;
+    deno_fmt = deno;
+    dioxus = dioxus-cli;
+    inherit (python313Packages) docformatter;
+    elm_format = elmPackages.elm-format;
+    erb_format = rubyPackages.erb-formatter;
+    fish_indent = fishMinimal;
+    forge_fmt = foundry;
+    format-dune-file = dune_3;
+    gdformat = gdtoolkit_4;
+    gofmt = go;
+    goimports = gotools;
+    hcl = hclfmt;
+    inherit (haskellPackages) hindent;
+    html_beautify = nodePackages.js-beautify;
+    inherit (rubyPackages) htmlbeautifier;
+    hurlfmt = hurl;
+    js_beautify = nodePackages.js-beautify;
+    jsonnetfmt = jsonnet;
+    inherit (texlive.pkgs) latexindent;
+    lua-format = luaformatter;
+    mago_format = mago;
+    mago_lint = mago;
+    markdownlint = markdownlint-cli;
+    mix = beamMinimal28Packages.elixir;
+    nginxfmt = nginx-config-formatter;
+    nimpretty = nim;
+    nixpkgs_fmt = nixpkgs-fmt;
```

```suggestion
    inherit nixfmt;
    nixpkgs_fmt = nixpkgs-fmt;
```
? I see it in the list at https://github.com/nix-community/nixvim/pull/3682/files#diff-e933535c9f5bd07146c4b67a8f94982c4d4f24cb3e3f70db608b8877b65be9ecR136 but don't see it here.

would be nice to have a test that everything from our generated list is accounted for, i believe we do something like that in lsp servers?

---

The conform name is identical to the nix package so the package is found without an explicit mapping.

The test i wrote accounts for every formatter, there just is this defaulting behaviour.

---

Sorry, just briefly skimmed and must have missed how it was being tested to account for that.